README.md

```diff
@@ -143,6 +143,17 @@ This information is passed along when the library makes calls to the Stripe
 API. Note that while `Name` is always required, `Url` and `Version` are
 optional.
 
+### Beta SDKs
```

Comment:
Can we reuse the content from https://github.com/stripe/stripe-node#beta-version which also talks about how to install the beta SDK or identify the beta SDK?

Reply:
Done.